server/app/api.py

```diff
@@ -1043,7 +1043,8 @@ def submit(self, user, assignment, messages, submit, submitter=None):
         submission = self.db.create_submission(user, valid_assignment,
                                                messages, submit, submitter)
         return (201, 'success', {
-            'key': submission.key.id()
+            'key': submission.key.id(),
+            'course': valid_assignment.course.id()
```

Comment:
@Some3141 @albert12132  - Is there a reason to not send the full URL and have the client display that url?